src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -686,16 +680,27 @@ class HttpGoogleServicesDAO(
     val projectNameString = projectName.value
     for {
       _ <- retryWhen500orGoogleError(() => {
-      executeGoogleRequest(billingManager.projects().updateBillingInfo(s"projects/${projectName.value}", new ProjectBillingInfo().setBillingEnabled(false)))
+        executeGoogleRequest(billingManager.projects().updateBillingInfo(s"projects/${projectName.value}", new ProjectBillingInfo().setBillingEnabled(false)))
       })
       - <- retryWhen500orGoogleError(() => {
-        executeGoogleRequest (resMgr.projects ().delete (projectNameString))
+        executeGoogleRequest(resMgr.projects().delete(projectNameString))
       })
     } yield {
       // nothing
     }
   }
 
+  def projectUsageExportBucketName(projectName: RawlsBillingProjectName): String = {
+    s"${projectName.value}-usage-export"
+  }
+
+  override def setProjectUsageExportBucket(projectName: RawlsBillingProjectName): Future[Try[Unit]] = {
+    val credential = getBillingServiceAccountCredential
+    val computeManager = getComputeManager(credential)
+
+    toFutureTry(Future(blocking(executeGoogleRequest(computeManager.projects().setUsageExportBucket(projectName.value, new UsageExportLocation().setBucketName(projectUsageExportBucketName(projectName)).setReportNamePrefix("usage"))))))
+  }
```

Comment:
I like this line of questioning, since `Unit` is a pretty unhelpful value, but I think the answer may be beyond my Scala-Fu and it is consistent with many of the other methods in this class. Maybe you had something in mind?